src/Benchmarks/Program.cs

```diff
@@ -50,8 +50,15 @@ public static void Main(string[] args)
                     .AddSingleton<Scenarios>()
                 );
 
+            var threadPoolDispatching = true;
```

Comment:
This should be a nullable bool to distinguish between `true`, `false`, and `Not Set`.  If the value is `Not Set`, then neither `options.ThreadCount` nor `internalOptions.ThreadPoolDispatching` should be modified.

This is a principle I've tried to maintain throughout Benchmarks.  If a Benchmarks config is not set, then framework defaults should not be changed.